packit/cli/init.py

```diff
@@ -67,7 +68,7 @@ def generate(path_or_url, force):
         "downstream_package_name": path_or_url.repo_name,
     }
 
-    generate_config(
+    init_config(
```

Comment:
This can probably stay the same. Init can have in the future more functionality, but the `generate_config` describes the function well.

Reply:
Yes, I will change it to generate